idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -0,0 +1,411 @@
+                            doc="Impeller work coefficient",
+                            bounds=(0.001,1))
+        self.mu_p_v = Var(self.flowsheet().config.time,
+                            initialize = 0.60,
+                            doc="Polytropic head coefficient vaned diffuser",
+                            bounds=(0.001,1))        
+        self.mu_p = Var(self.flowsheet().config.time,
+                            initialize = 0.60,
+                            doc="Polytropic head coefficient",
+                            bounds=(0.001,1))
+        self.eff_p_v = Var(self.flowsheet().config.time,
+                            initialize=0.85,
+                            doc="Polytropic stage efficiency vaned diffuser",
+                            bounds=(0.001,1))
+        self.eff_p = Var(self.flowsheet().config.time,
+                            initialize=0.85,
+                            doc="Polytropic stage efficiency",
+                            bounds = (0.001, 1.0))       
+        
+        self.delta_enth_polytropic = Var(self.flowsheet().config.time,
+                            initialize=2548.5,
+                            doc="Polytropic enthalpy change [J/mol]",
+                            within=NonNegativeReals)
+        
+        self.U2 = Var(self.flowsheet().config.time,
+                            initialize = 315, 
+                            doc="Impeller tip speed", 
+                            bounds= (0,4000)) # upper: 4000, lower: 0 // m/s
+        self.Ma = Var(self.flowsheet().config.time,
+                            initialize = 0.1,
+                            doc = 'Rotational Ma number',
+                            bounds = (0,5))
+        self.rspeed = Var(self.flowsheet().config.time,
+                            initialize = 1500,
+                            doc = 'Rotation speed of the impeller',
+                            bounds = (0.5, 8000))      
+        self.r2 = Var(initialize= 0.075, 
+                            doc="Impeller tip radius")  
+        
+        ########################################################     
+        
+        self.dp = Param(default= 1, within=Reals, doc='a pressure difference for finite difference derivative calculation')
+        self.diffuser_type = Param(default=1.0, 
+                                    mutable=True,
+                                    doc="1 for vane diffuser, 2 for vaneless diffuser")
+        self.impeller_type = Param(default=2.0,
+                                    mutable=True,
+                                    doc="1 for covered impeller, 2 for open impeller")
+        self.efficiency_mech = Param(default= 0.97,
+                                    doc="Compressor mechanical efficiency")
+        self.eff_drive = Param(default= 1.0,
+                                doc="Compressor driver efficiency")        
+
+        ############################################################ 
+        # Fix variables 
```

Comment:
This is changing the initial value, not fixing the value.

Reply:
True, and if that's what you want to do, better off using the initialize argument when you define the Var.

Reply:
These variables should be changed. I have deleted this line.